File: 0125-valid-palindrome/0125-valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {
    s=s.toLowerCase();
    let b="";
    for(let i=0;i<s.length;i++)
    {
        if((s[i]>='a' && s[i]<='z') || (s[i]>='0' && s[i]<='9')) b+=s[i];
    }
    s=b.split("");
    s=s.reverse();
    s=s.join("");
    return b==s;
};

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string a="";
        for(char c:s){
            if(c>='A' && c<='Z'){
                c=tolower(c);
                a+=c;
            }
            else if((c>='a' && c<='z') || (c>='0' && c<='9')) a+=c;
        }
        cout<<a;
        string b="";
        for(int i=a.length()-1;i>=0;i--)
        {
            b+=a[i];
        }
        return a==b;
    }
};